### PR TITLE
Add an option to disable double-click text selection

### PR DESCRIPTION
**Pull request by JDiPierro**
Added an option to disable SyntaxHighlighter's `quickCode` setting.

### CODE REVIEW

File: options.html [hunk 1]
```diff
@@ -31,6 +31,8 @@
 			<div class="line number21 index20 alt2">21</div>
 			<div class="line number22 index21 alt1">22</div>
 			<div class="line number23 index22 alt2">23</div>
+			<div class="line number22 index23 alt1">24</div>
+			<div class="line number22 index23 alt1">25</div>
```
Comment (deiga) [inline options.html]: It would seem these classnames are not correct

Comment (JDiPierro) [inline options.html]: Fixed; thanks.

**Comment by ajmath** [comment]
published in 3.0.132

**Comment by JDiPierro** [comment]
Ahhh, awesome :D ! Thank you

**Pull request merged**